File: archives/10101.py
a = []
for _ in range(3):
    a.append(int(input()))

if (a == [60, 60, 60]):
    print("Equilateral")
elif (sum(a) != 180):
    print("Error")
else:
    if (a[0] == a[1] or a[0] == a[2] or a[1] == a[2]):
        print("Isosceles")
    else:
        print("Scalene")

File: archives/cpp/10101.cpp
#include <bits/stdc++.h>
#define FOR(i, start, end) for (int i = start; i < end; i++)
#define endl "\n"
#define INT_INF 2147483647
#define MOD 1
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int a, b, c; cin >> a; cin >> b; cin >> c;
    if (a == 60 && b == 60 && c == 60) {
        cout << "Equilateral";
    } else if (a+b+c != 180) {
        cout << "Error";
    } else if (a==b || b==c || a==c) {
        cout << "Isosceles";
    } else {
        cout << "Scalene";
    }

    return 0;
}